database/schema.sql

```diff
@@ -0,0 +1,23 @@
+DROP DATABASE IF EXISTS related_products;
+
+CREATE DATABASE related_products;
+
+USE related_products;
+
+CREATE TABLE products (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(200),
+  price VARCHAR(20),
+  style_ids JSON,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE styles (
+  id INT,
+  name VARCHAR(200),
+  img_url VARCHAR(500)
+);
+
+/*  To execute:
+ *    mysql -u root < database/schema.sql

```

Comment:
This is probably better fit for your `README.md` instead.

Reply:
Good point, I'll create one next.